src/activity_type.cpp

```diff
@@ -76,8 +79,9 @@ void activity_type::load( const JsonObject &jo )
             }
         }
 
-        if( c_moves.has_array( "qualities" ) ) {
-            result.tools_affected_ = true;
+        if( c_moves.has_bool( "qualities" ) ) {
+            assign( jo, "qualities", result.tools_affected_, false );
+        } else if( c_moves.has_array( "qualities" ) ) {
```

Comment:
```suggestion
        } else if( c_moves.has_array( "qualities" ) ) {
            result.tools_affected_ = true;
```
it looks like this got accidentally omitted?